src/manager.cpp

```diff
@@ -112,15 +126,68 @@ void Manager::SetTimerToDetectSVPDOnDbus()
 }
 #endif
 
-void Manager::updateKeyword(const types::Path i_path,
-                            const types::VpdData i_data, const uint8_t i_target)
+void Manager::updateKeyword(types::Path i_path, const types::VpdData i_data)
 {
-    // Dummy code to supress unused variable warning.
-    std::cout << "\nFRU path " << i_path;
-    std::cout << "\nData " << i_data.index();
-    std::cout << "\nTarget = " << static_cast<int>(i_target);
+    types::PathCollection l_fruPaths;
```

Comment:
Why this api is tightly coupled with config JSON?
JSON can be used to additional information, but in case not defined the functionality should still be provided and allow updating of EEPROM kwd value.
Please revisit the design here.